dev/prod_builders.json

```diff
@@ -450,6 +450,240 @@
          "task_name": "mac_build_aar_module_test",
          "flaky": false
       },
+      {
+         "name": "Mac_android android_plugin_example_app_build_test",
+         "repo": "flutter",
+         "task_name": "mac_android_android_plugin_example_app_build_test",
+         "flaky": false
+      },
+      {
+         "name": "Mac_android android_semantics_integration_test",
+         "repo": "flutter",
+         "task_name": "mac_android_android_semantics_integration_test",
+         "flaky": true
```

Comment:
actually, can you mark this not flaky? this should pass at ToT

Reply:
done